src/s3ql/backends/gs.py

```diff
@@ -200,7 +201,7 @@ def _get_conn(self):
         conn = HTTPConnection(
             self.hostname, self.port, proxy=self.proxy, ssl_context=self.ssl_context
         )
-        conn.timeout = int(self.options.get('tcp-timeout', 20))
+        conn.timeout = int(self.options.get('tcp-timeout', 20)) * 1000
```

Comment:
Does it make sense to rename this `timeout_ms`?

Reply:
Yes, thanks!